instagram-app/src/App.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import "./App.css";
+import dummyData from "./dummy-data.js";
+import SearchBar from "./components/SearchBar/SearchBar";
+import PostContainer from "./components/PostContainer/PostContainer";
+
+class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      data: [],
+      filteredPosts: []
+    };
+  }
+
+  changeHandler = element => {
+    this.setState({
+      [element.target.name]: element.target.value
+    });
+  };
+
+  componentDidMount() {
+    this.setState({
+      data: dummyData
+    });
+  }
+
+  searchFilter = element => {
```

Comment:
You can technically name function parameters whatever you want, but we should practice naming them semantically. searchFilter is being called when searchBar changes, so what should be passed into it? https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Events#Event_objects . Check the rest of the methods you are calling on events, do they follow this convention?